lib/fog/ovirt/compute/v4.rb

```diff
@@ -80,6 +80,7 @@ def get_attr_value(value, opts)
               value.address
             when OvirtSDK4::Cpu
               opts[:cores] = value.topology.nil? ? nil : value.topology.cores
+              opts[:sockets] = value.topology.nil? ? nil : value.topology.sockets
```

Comment:
We can use `#try` here:

``` ruby
opts[:cores] = value.topology.try(:cores)
opts[:sockets] = value.topology.try(:sockets)
```

Reply:
try is rails function, it can't be used in fog-ovirt.